src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -372,9 +347,9 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                 FunctionBinding.ConvertStreamToValue((Stream)input, dataType, ref input);
             }
 
-            Utility.ApplyBindingData(input, binder.BindingData);
-            var bindingData = NormalizeBindingData(binder.BindingData);
-            bindingData["invocationId"] = functionExecutionContext.InvocationId.ToString();
+            Utility.ApplyBindingData(input, invocationContext.Binder.BindingData);
```

Comment:
Same thing about relying on the `Binder` since it is treated as optional in the base class. If we expect the infrastructure to always request/require a binder, should it not be treated as optional?